Gemfile

```diff
@@ -26,15 +26,23 @@ gem 'bootsnap', '>= 1.4.4', require: false
 # gem 'rack-cors'
 
 group :development, :test do
-  # Call 'byebug' anywhere in the code to stop execution and get a debugger console
-  gem 'byebug', platforms: [:mri, :mingw, :x64_mingw]
+  gem 'byebug', platforms: %i[mri mingw x64_mingw]
+  gem 'factory_bot_rails', '~> 6.2'
+  gem 'ffaker', '~> 2.18'
+  gem 'pry-byebug', '~> 3.9'
+  gem 'pry-rails', '~> 0.3.9'
+  gem 'rspec-rails', '~> 5.0', '>= 5.0.1'
+  gem 'rubocop', '~> 1.18', '>= 1.18.3'
```

Comment:
Please read https://confluence.softserveinc.com/pages/viewpage.action?pageId=447697648, you should use all stack from linter paragraph.